todo_app/data/roleChecker.py

```diff
@@ -0,0 +1,11 @@
+from flask_login import current_user
+from flask import redirect
+import os
+
+def ReadWriteNeeded(func):
+    def wrapper(*args, **kwargs):
+        if current_user.role == 'Read/Write' or os.getenv('LOGIN_DISABLED') == 'True':
```

Comment:
You dodge it here because your tests don't check any endpoints covered by this, but if they did I think this would error because when `LOGIN_DISABLED` is turned on the current_user isn't set. Switching the order of these would mean that in that scenario it doesn't bother to try run the role check so you get away with it:
```suggestion
        if os.getenv('LOGIN_DISABLED') == 'True' or current_user.role == 'Read/Write':
```

You can test that by switching on LOGIN_DISABLED and then trying to visit your site in a new (e.g. incognito) browser and creating an item